src/pages/settings/AboutPage/TroubleshootPage.tsx

```diff
@@ -0,0 +1,120 @@
+import * as App from '@userActions/App';
+import * as Report from '@userActions/Report';
+import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {OnyxKey} from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
+
+const keysToPreserve: OnyxKey[] = [
+    ONYXKEYS.ACCOUNT,
+    ONYXKEYS.IS_CHECKING_PUBLIC_ROOM,
+    ONYXKEYS.IS_LOADING_APP,
+    ONYXKEYS.IS_SIDEBAR_LOADED,
+    ONYXKEYS.MODAL,
+    ONYXKEYS.NETWORK,
+    ONYXKEYS.SESSION,
+    ONYXKEYS.SHOULD_SHOW_COMPOSE_INPUT,
+    ONYXKEYS.NVP_TRY_FOCUS_MODE,
+    ONYXKEYS.PREFERRED_THEME,
+    ONYXKEYS.NVP_PREFERRED_LOCALE,
+];
+
+type BaseMenuItem = {
+    translationKey: TranslationPaths;
+    icon: React.FC<SvgProps>;
+    action: () => void;
+};
+
+function TroubleshootPage() {
+    const {translate} = useLocalize();
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {isProduction} = useEnvironment();
+    const [isConfirmationModalVisible, setIsConfirmationModalVisible] = useState(false);
+
+    const menuItems = useMemo(() => {
+        const baseMenuItems: BaseMenuItem[] = [
+            {
+                translationKey: 'initialSettingsPage.troubleshoot.clearCacheAndRestart',
+                icon: Expensicons.RotateLeft,
+                action: () => setIsConfirmationModalVisible(true),
+            },
+        ];
+
+        return baseMenuItems.map((item) => ({
+            key: item.translationKey,
+            title: translate(item.translationKey),
+            icon: item.icon,
+            onPress: item.action,
+        }));
+    }, [translate]);
+
+    return (
+        <IllustratedHeaderPageLayout
+            title={translate('initialSettingsPage.aboutPage.troubleshoot')}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_ABOUT)}
+            backgroundColor={theme.PAGE_THEMES[SCREENS.SETTINGS.TROUBLESHOOT].backgroundColor}
+            illustration={LottieAnimations.Desk}
+            testID={TroubleshootPage.displayName}
+        >
+            <View style={[styles.settingsPageBody, styles.ph5]}>
+                <Text style={[styles.textHeadline, styles.mb1]}>{translate('initialSettingsPage.aboutPage.troubleshoot')}</Text>
+                <Text style={styles.mb4}>
+                    <Text>{translate('initialSettingsPage.troubleshoot.description')}</Text>{' '}
+                    <TextLink
+                        style={styles.link}
+                        onPress={() => Report.navigateToConciergeChat()}
+                    >
+                        {translate('initialSettingsPage.troubleshoot.submitBug')}
+                    </TextLink>
+                </Text>
+            </View>
+            <MenuItemList
+                menuItems={menuItems}
+                shouldUseSingleExecution
+            />
+            {/* Enable additional test features in non-production environments */}
```

Comment:
Not a blocker:

From David's initial proposal https://expensify.slack.com/archives/C01GTK53T8Q/p1703982689203629, I thought we would also show this on PROD.

> Have a "secret knock" to make this page appear on production builds (eg, triple-tap the E icon at the top of the About page), which enables and navigates to it

Did we decide to show the TestToolMenu only on non-prod env? This isn't necessary for now but just wanted to confirm.

Reply:
Good point, I left the logic for displaying `TestToolMenu` unchanged, not sure if e.g. `Use staging server` option should be visible on prod? 🤔